README.md

```diff
@@ -1 +1,12 @@
 # Git secrets check
+
+This repository is a Jenkinsfile which periodically runs git secrets against a test file. The test file contains 3 fake secrets.
+
+* A fake AWS account number
```

Comment:
Should this be access key rather than account number?

Reply:
Yep good spot, cheers.